docs/source/pipelines.rst

```diff
@@ -118,38 +119,106 @@ Key concepts
 
 Usage
 -----
+Development environments
+~~~~~~~~~~~~~~~~~~~~~~~~
+We recommend to develop models with MLflow Pipelines with one of the following environment settings:
 
-The general workflow for using MLflow Pipelines is as follows:
+[**Databricks**]
+  - Edit YAML config and Python files in Databricks Repos. Open separate browser tabs for each
+    file module that you wish to modify.
+  - Use ``notebooks/databricks.py`` as the driver to run pipeline steps and inspect its output.
+  - Pin the workspace browser for easy file navigation.
+
+  .. image:: _static/images/pipelines_databricks_ui.png
+    :width: 60%
+
+[**Local with Jupyter Notebook**]
+  - Use ``notebooks/jupyter.ipynb`` as the driver to run pipeline steps and inspect its output.
+  - Use commandline such as ``mlflow pipelines run --profile local`` to run the entire pipeline
+    after done with model quality iteration.
```

Comment:
Where would I run this command? At the project root?

Reply:
Hmmm, why would a user want to run the entire pipeline once after model iteration? Is it to see if the pipeline can be run as a job?

Reply:
Not sure what this means:

```
after done with model quality iteration.
```

Can we clarify?

Reply:
Updated. I'm trying to provide alternative ways to run the pipeline (instead of through the notebook).